React/my-app/src/components/MultiStateButton/styles.css

```diff
@@ -0,0 +1,51 @@
+.button_box {
```

Comment:
We use camel case for css classes.
If you are using snake case in all classes we can keep it.